### DISCUSSION TITLE
run pnpm install will update lockfile automatically

### DISCUSSION THREAD

**Discussion by lingxiao-Zhu** [pnpm/pnpm#5827] [Q&A]
I think this will make prod problems，or do I need add some params？

**Comment by zkochan** [comment]
The lockfile changes if there are changes in the `package.json` files of your workspace (or in the `package.json` file of your project). If there were no changes in `package.json` then it is a bug.

You can also run `pnpm install --frozen-lockfile`. In this case pnpm will fail if it cannot run installation without making changes to the lockfile.
